spec/unit/puppet_x/chocolatey/chocolatey_install_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'spec_helper'
+require 'puppet_x/chocolatey/chocolatey_install'
+
+describe 'Chocolatey Install Location' do
+
+  context 'on Windows', :if => Puppet::Util::Platform.windows? do
+
+    it "should return install path from registry if it exists" do
+      expected_value = 'C:\somewhere'
+      Win32::Registry.any_instance.expects(:[]).with('ChocolateyInstall').returns(expected_value)
+
+      PuppetX::Chocolatey::ChocolateyInstall.install_path.must == expected_value
+    end
+
+    it "should return the default install path if environment variable does not exist" do
+      expected_value = 'C:\ProgramData\chocolatey'
+      Win32::Registry.any_instance.expects(:[]).with('ChocolateyInstall').raises(Win32::Registry::Error.new(2), 'file not found yo')
+
+      PuppetX::Chocolatey::ChocolateyInstall.install_path.must == expected_value
+    end
+  end
+
+  context 'on Linux', :if => Puppet.features.posix? do
+    it "should return the default install path on a non-windows system" do
+      PuppetX::Chocolatey::ChocolateyInstall.install_path.must == 'C:\ProgramData\chocolatey'
```

Comment:
This seems counter-intuitive.  On a linux system wouldn't the install path be nil or empty?